odoo/addons/base/models/ir_attachment.py

```diff
@@ -345,6 +345,9 @@ def _postprocess_contents(self, values):
                     # Catch error during test where we provide fake image
                     # raise UserError(_("This file could not be decoded as an image file. Please try with a different file."))
                     _logger.info('Post processing ignored : %s', e)
+                except Image.DecompressionBombError as e:
+                    e.sentry_ignored = True
+                    raise
```

Comment:
Why not catching `DecompressionBombWarning` as well ?

Reply:
@yelizariev 
Changes have been done.
Thanks!